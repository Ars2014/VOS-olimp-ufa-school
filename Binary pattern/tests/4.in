80219
14267
14267
520585727
520585727
