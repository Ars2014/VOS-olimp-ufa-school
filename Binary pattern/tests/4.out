Pattern
No
No
No
No
